tox.ini

```diff
@@ -6,7 +6,7 @@ no_package = True
 env_list = lint, unit
 
 [vars]
-src_path = {tox_root}/prometheus_opensearch_dashboards_exporter
+src_path = {tox_root}/src/prometheus_opensearch_dashboards_exporter
```

Comment:
I think this should just be ```{tox_root}/src```.  prometheus_opensearch_dashboards_exporter is a package within the src_path, and Python should be able to pick it up if we give the src directory here.